Cubical/Functions/Image.agda

```diff
@@ -0,0 +1,75 @@
+{-# OPTIONS --safe #-}
+module Cubical.Functions.Image where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Equiv.Fiberwise
+open import Cubical.Foundations.Univalence
+
+open import Cubical.Functions.Logic
+open import Cubical.Functions.Embedding
+open import Cubical.Functions.Surjection
+
+open import Cubical.HITs.PropositionalTruncation as PT
+open import Cubical.Data.Sigma
+
+private variable
+  ℓ ℓ' : Level
+  A B : Type ℓ
+
+module _ (f : A → B) where
+  isInImage : B → Type _
+  isInImage y = ∃[ x ∈ A ] f x ≡ y
+
+  image : Type _
```

Comment:
Shouldn't this be `Image` as it's a type?

Reply:
changed